server/tests/audit_math/test_minerva2.py

```diff
@@ -0,0 +1,239 @@
+from r2b2.minerva2 import Minerva2 as Providence
```

Comment:
Is this naming ever likely to change up stream? Not that it matters since we'll pin the library, but in the interest of killing Minerva2 references...

Reply:
I brought it up with Oliver and he said he'll get around to fixing it! The dependency is pinned but when `r2b2` updates we can fix it : )